Tiltfile

```diff
@@ -157,21 +157,6 @@ def capz():
         deps = ["api", "cloud", "config", "controllers", "exp", "feature", "pkg", "go.mod", "go.sum", "main.go"]
     )
 
-    k8s_resource('capz-controller-manager:deployment:capz-system', objects=[
```

Comment:
why remove these?

Reply:
tilt was complaining about this line saying I doesn't have a a deployment, don't know if I need to define it somewhere else